tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -258,5 +258,31 @@
       _dd.tracer_kr: 1.0,
       _sampling_priority_v1: 1.0
     }
+  },
+  {
+    TraceId: Id_15,
+    SpanId: Id_16,
+    Name: AdditionalActivitySource.internal,
+    Resource: Transform,
+    Service: CustomServiceName,
+    Type: custom,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: AdditionalActivitySource,
+      otel.library.version: ,
```

Comment:
We probably shouldn't add this tag if it's empty should we? It doesn't appear to get added to the other (Otel) spans?